src/js/page/ui/copy-button.js

```diff
@@ -23,6 +23,12 @@ export default class CopyButton extends FloatingActionButton {
 
   _onClick(event) {
     super._onClick(event);
+    this.getCopyText();
+  }
+
+  getCopyText() {
```

Comment:
Could you rename this to indicate it performs an action? Right now it reads like it "gets the text to be copied". Maybe `copyText`?